pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -684,9 +690,9 @@ private static boolean existsParticipant(KitRequestShipping kit, ElasticSearchPa
     }
 
     public static Map<String, List<KitRequestShipping>> getAllKitRequestsByRealm(@NonNull String realm, String target, String kitType,
-                                                                                 boolean getAll) {
+                                                                                 Integer kitTypeId, boolean getAll) {
```

Comment:
Perhaps the `kitType` parameter should be name `kitTypeName`, now that we have a `kitTypeId`?